contributing/testing.md

```diff
@@ -18,6 +32,28 @@ You can install backstop and set up the demo app by running the following from w
 ./bin/setup
 ```
 
+#### Compile packs
+
+You will need to compile any changes to the assets for the test environment:
+
+```sh
+RAILS_ENV=test bin/webpacker:compile
```

Comment:
I think this might need to be `… bin/rails webpacker…`.

Running it for me fails though which is frustrating. 

I first got:

```
ModuleNotFoundError: Module not found: Error: Can't resolve '@citizensadvice/design-system/lib/header' in '/Users/tomhipkin/sites/citizens-advice/tahb/design-system/demo/app/frontend/packs'
    at /Users//sites/citizens-advice/tahb/design-system/demo/node_modules/webpack/lib/Compilation.js:925:10
```

I also saw a warning `Warning: you are using an unstable release of Node.js (v17.7.2)` so tried to bump my node to 18 but got a new error:

```
Compiling...
Compilation failed:
node:internal/crypto/hash:67
  this[kHandle] = new _Hash(algorithm, xofLen);
                  ^
Error: error:0308010C:digital envelope routines::unsupported
    at new Hash (node:internal/crypto/hash:67:19)
```

Reply:
`bin/webpack-dev-server` doesn't work either. I hate webpacker.

Reply:
The first error smells like you have not run `npm run build` from the root before going into `demo` and running `RAILS_ENV=test bin/webpacker:compile`. To be honest, I am not in love with how this works, but it's not ours so I decided not to wade in with opinions (too much!)

In terms of Node version, I treat it like ruby, the (docs say 14)[https://github.com/citizensadvice/design-system/blob/main/contributing/setup.md#prerequisites], so I set that up,

Reply:
Everything passes now. Though I did have to change it to `RAILS_ENV=test bin/rails webpacker:compile`

Yeah that's right had to go back up and run npm build. Despite the `**note**` I was trying to earlier `compile` step since I knew I had never run it. I wonder if that would be read first?

"If you have not done so, run npm run build from the root and then come back to /demo" or similar?

Reply:
🤦 I've just seen what you mean!! Yep `bin/rails webpacker:compile` sorry!